Cargo.toml

```diff
@@ -83,7 +83,7 @@ surrealdb = { version = "1", path = "lib", features = [
 	"protocol-http",
 	"protocol-ws",
 	"rustls",
-	"sql2",
+	"sql2"
```

Comment:
The comma is fine. Can you revert this commit? I will approve it on behalf of security.